grails-app/views/purchaseOrderWorkflow/purchaseOrder/showOrderItems.gsp

```diff
@@ -523,6 +527,29 @@
           });
         }
 
+        function updateOrderItems() {
```

Comment:
@DominikaHatala I approved it, but lets adjust this naming. Because now we have loadorderItems and updateorderitems and the naming is very vague. Update this to something more descriptive.

Reply:
@DominikaHatala i guess the best case would if the items were updated for select during loadOrderItems, but if its problematic, then let's distinghuish it with proper naming.